### DISCUSSION TITLE
Jamulus and Chat

### DISCUSSION THREAD

**Discussion by BlindRyan** [community/community#28520] [Other Feature Feedback, Questions, & Ideas]
Hi as someone who is totally blind when jamming with others if ### so
meone keeps throwing messages into chat it gets pretty disruptive so could we have a way to turn the chat off if we wanted to? #27279

**Comment by radar-blip** [comment]
You can unsubscribe to that whole thread. Through notifications or the thread itself. That should do it.
There should be an 'Unsubscribe button to your right.


![subs](https://user-images.githubusercontent.com/39196767/183223218-89e0ea3a-ff30-41b3-820a-c22f940725c3.png)

**Comment by BlindRyan** [comment]
Thanks for the reply but what I meant is being able to disable Temporarity the chat function that is built into Jamulus some musicians can chat with each other or post links to guitar tabs and such

**Reply by radar-blip** [reply]
Cant help you with Jamulus, not familiar. 
Good luck and have a nice flight!

**Comment by airtower-luna** [comment]
Hi, it looks like Jamulus is a software hosted on GitHub, I think this is the repository: https://github.com/jamulussoftware/jamulus

In that case you should create an issue or discussion *in that repository*, so it gets to people who know the software.

**Comment by BlindRyan** [comment]
Thanks I will do that

**Comment by ann0see** [comment]
@BlindRyan I‘m one of the jamulus maintainers. Feel free to open an issue on our repository as linked above. I think the chat feature can’t be disabled at the moment.
